js/src/personal/riyuan/components/navigation/Navigation.tsx

```diff
@@ -55,16 +58,16 @@ const NavItem = ({
   isOpen: boolean
 }) => (
   <div className={styles.navLink}>
-    <Link
-      to={`#${title}`}
+    <a
```

Comment:
curious, what the difference is between Link, a, and Anchor that made you pick <a/> here?

Reply:
I change it back to a tag, cause the link isnt working